### PR TITLE
[CI] adding win fulldebug build for Core

### PR DESCRIPTION
**Pull request by philbucher**
see https://github.com/KratosMultiphysics/Kratos/pull/6985#issuecomment-635892641

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -144,3 +144,70 @@ jobs:
...
+        del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeCache.txt"
+        del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeFiles"
+
+        cmake                                                ^
+          -G"Visual Studio 16 2019"                          ^
+          -H"%KRATOS_SOURCE%"                                ^
+          -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"             ^
+          -DBOOST_ROOT="%BOOST_ROOT_1_72_0%"                 ^
+          -DINSTALL_RUNKRATOS=OFF                            ^
+          -DCMAKE_CXX_FLAGS="/Od                             ^
```
Comment (philbucher) [inline .github/workflows/ci.yml]: @roigcarlo what to do here? Remove?

Comment (roigcarlo) [reply .github/workflows/ci.yml]: You can leave it, fulldebug also adds that. It was on the regular one so it does not enable optimization which make is took much more time to compiple

**Comment by philbucher** [comment]
ok this takes waaaaaaaaaaaaaaay too much time, both the cpp-tests and the python tests
=> seems like this is not feasible, but I am no Win-expert

**Comment by philbucher** [comment]
closing as waaaaaaaaaaaaaaaaaaaaaaaaay too slow

**Pull request closed without merging**